backend/routers/user.py

```
@@ -14,6 +32,89 @@ class ExportFormat(str, Enum):
+    for job_id, job in list(export_jobs.items()):
+        created_at_str = job.get("created_at")
+        completed_at_str = job.get("completed_at")
+        reference_time_str = completed_at_str or created_at_str
+
+        if not reference_time_str:
+            continue
+
+        try:
+            reference_time = datetime.fromisoformat(reference_time_str)
+        except Exception:  # noqa: BLE001
+            continue
```

Ruff (0.15.2)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)